--- a/config.def.h
+++ b/config.def.h
@@ -3,11 +3,11 @@
 static const char *group = "nogroup";
 
 static const char *colorname[NUMCOLS] = {
-	[INIT] =   "black",     /* after initialization */
-	[INPUT] =  "#005577",   /* during input */
-	[INPUT_ALT] = "#227799", /* during input, second color */
-	[FAILED] = "#CC3333",   /* wrong password */
+	[INIT] =   "#000000",       /* after initialization */
+	[INPUT] =  "#007700",       /* during input */
+	[INPUT_ALT] = "#229900",    /* during input, second color */
+	[FAILED] = "#FF0000",       /* wrong password */
 };
 
 /* treat a cleared input like a wrong password (color) */
-static const int failonclear = 1;
+static const int failonclear = 0;
